monitoring/src/main/scala/akkaviz/server/WebSocketRepl.scala

```diff
@@ -172,7 +164,7 @@ trait WebSocketRepl {
      * Runs your code with supplied environment installed.
      * After execution of supplied code block will restore original environment
      */
-    def withEnvironment(env: Environment)(code: ⇒ Any): Any = {
+    def withEnvironment(env: Environment)(code: => Any): Any = {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You should not name methods after their defining object](https://www.codacy.com/app/lustefaniak/akka-viz/file/2205313427/issues/source?bid=3121282&fileBranchId=3170260#l167)